taskcluster/ci/generate-screenshots/kind.yml

```diff
@@ -5,46 +5,22 @@
 loader: taskgraph.loader.transform:loader
 
 transforms:
-    - ffios_taskgraph.transforms.screenshots:transforms
+    - ffios_taskgraph.transforms.bitrise:transforms
     - ffios_taskgraph.transforms.secrets:transforms
     - taskgraph.transforms.job:transforms
     - taskgraph.transforms.task:transforms
 
+kind-dependencies:
+    - build
 
 job-defaults:
+    bitrise-workflow: jlorenzo_L10nScreenshotsTests
+    build-derived-data-path: {artifact-reference: '<build/public/build/target.zip>'}
```

Comment:
This line is some taskgraph magic which gives the full URL, once interpolated. This way, we tell bitrise to download the zip file from a full public URL.

It knows what `build` is thanks to:

```yaml
    dependencies:
        build: build-screenshots
```

down below.